src/wfutil.c

```diff
@@ -1617,3 +1619,78 @@ BOOL TypeAheadString(WCHAR ch, LPWSTR szT)
 
    return ich != 0;
 }
+
+// Search the list of slots if path would be parent of already existing drive == circularity
+BOOL FindUNCLoop(LPCTSTR path)
+{
+   for (DWORD dwDriveIndex = OFFSET_UNC; dwDriveIndex < MAX_DRIVES; ++dwDriveIndex) {
+      if (aDriveInfo[dwDriveIndex].szRoot[0] && StrStrIW(aDriveInfo[dwDriveIndex].szRoot, path))

```

Comment:
Using shell functions here seems strange.  Is this because there's no case-insensitive wcsistr type thing?

Reply:
Correct. Same in treectl:1416